### ISSUE TITLE
Datetimes fail to parse

### ISSUE THREAD

**Issue by evamaxfield** [AllenCellModeling/czi-to-ome-xslt#11]
```python
In [4]: re = AICSImage("overview-translated.ome.tiff")
OME XML validation failed
failed validating '2016-03-11T10:23:44.925154UTC' with XsdAtomicBuiltin(name='xs:dateTime'):

Reason: attribute StartTime='2016-03-11T10:23:44.925154UTC': Invalid datetime string '2016-03-11T10:23:44.925154UTC' for <class 'elementpath.datatypes.datetime.DateTime10'>

Schema:

  <xs:simpleType xmlns:hfp="http://www.w3.org/2001/XMLSchema-hasFacetAndProperty" xmlns:xs="http://www.w3.org/2001/XMLSchema" name="dateTime" id="dateTime">
    <xs:annotation>
    <xs:appinfo>
        <hfp:hasFacet name="pattern" />
        <hfp:hasFacet name="enumeration" />
        <hfp:hasFacet name="whiteSpace" />
        <hfp:hasFacet name="maxInclusive" />
        <hfp:hasFacet name="maxExclusive" />
        <hfp:hasFacet name="minInclusive" />
        <hfp:hasFacet name="minExclusive" />
        <hfp:hasProperty name="ordered" value="partial" />
        <hfp:hasProperty name="bounded" value="false" />
        <hfp:hasProperty name="cardinality" value="countably infinite" />
        <hfp:hasProperty name="numeric" value="false" />
      </xs:appinfo>
      <xs:documentation source="http://www.w3.org/TR/xmlschema-2/#dateTime" />
    </xs:annotation>
    <xs:restriction base="xs:anySimpleType">
      <xs:whiteSpace value="collapse" fixed="true" id="dateTime.whiteSpace" />
    </xs:restriction>
  </xs:simpleType>

Instance:

  <PlateAcquisition xmlns="http://www.openmicroscopy.org/Schemas/OME/2016-06" ID="PlateAcquisition:1" StartTime="2016-03-11T10:23:44.925154UTC" />

Path: /OME/Plate/PlateAcquisition
```

**Comment by MatteBailey** [comment]
Turns out this is actually a bug in `ome-types` (or one of the underlying libs): https://github.com/tlambert03/ome-types/issues/104. We should probably close this issue and open one in `AICSImageIO`.

**Comment by tlambert03** [comment]
fixed in https://github.com/tlambert03/ome-types/pull/106 ... just released [v0.2.9](https://pypi.org/project/ome-types/) that includes the fix.

note also, I fixed the issue that required pinning to xmlschema 1.4.1, so xmlschema is now only pinned to `>=1.4.1` for ome-types.  just a heads up in case if this will affect what you want to do in [your pins](https://github.com/AllenCellModeling/aicsimageio/blob/982efbb2abb855d41daee2f659c84e7c4be6d4db/setup.py#L90).

**Comment by MatteBailey** [comment]
Got it, thanks @tlambert03!
